core_data/src/main/java/com/trempel/core_network/bag_db/db/BagDbRepositoryImpl.kt

```diff
@@ -0,0 +1,35 @@
+package com.trempel.core_network.bag_db.db
+
+import android.database.sqlite.SQLiteConstraintException
+import javax.inject.Inject
+
+class BagDbRepositoryImpl @Inject constructor(
```

Comment:
Looks like we can have common Bag database and related classes in core module but repositories can be feature specific